src/openrct2/drawing/TTFSDLPort.cpp

```diff
@@ -995,6 +995,7 @@ void TTF_CloseFont(TTF_Font* font)
             fclose(font->src);
         }
         free(font);
+        delete font;
```

Comment:
This change is wrong, all of the allocations of font are done using `malloc`. Either you have to update them to use `new` or you have to leave as it was.

This way you're doing what's called as mismatched free()/delete. `malloc` always goes with `free` and `new` with `delete`, you shouldn't mix and match, especially double deletion like here.